### DISCUSSION TITLE
CensysNotFoundException (whois)

### DISCUSSION THREAD

**Discussion by AkikoOrenji** [censys/censys-python#55] [Q&A] [answered]
When doing "results = c.view("x.x.x.x")" i get the following exception......

Censys.exceptions.CensysNotFoundException: 404 (unknown): We don't know anything about the specified host.

I notice in the UI when i run the same query i get "**Basic Information**"  and all the whois information on the **WHOIS** tab. 
Can i get the information contained on these other UI pages via the censys API ?  

At the moment the exception says "We don't know anything about the specified host." which is not true as you have the ASN and other whois information stored.
  If i can't access this data via the API  this means i need to go and collect it elsewhere which is not great given we pay for censys and the data is available via the UI.

**Comment by thehappydinoa** [comment, marked as answer]
Hi there,

We do not provide whois data via API, we do however make it easy for people to query whois from their browser. Please let me know if you have any questions I can help answer.
